index.html

```diff
@@ -5,69 +5,98 @@
+          Currently, I’m the Web Manager for Humble & Fume Inc., where I work
+          cross functionally to maintain the company’s multiple B2B sites.
+          During my time here, I’ve boosted our SEO position on Google search,
+          have maintained and uploaded thousands of products (which includes
+          image editing, descriptions, option sets, etc), written the Standard
+          Operating Procedure for maintaining brand standards across web, and
+          have managed hundreds of successful sales.
+        </p>
+      </section>
 
-      <h4>skills</h4>
-      <ul>
-        <li>HTML</li>
-        <li>CSS</li>
-        <li>JavaScript</li>
-      </ul>
-    
-    </div>
+      <section>
+        <header>
+          <h2>&lt; my experience &gt;</h2>
+          <h4>Web Manager | humble + fume</h4>
+        </header>
+        <p>
+          I'm an experienced web product & content manager with a demonstrated
+          history of working in B2B and B2C marketplaces. I have an extensive
+          background in eCommerce product management—product descriptions,
+          metadata, SEO, lifecycle management, implementing sales/promotions on
+          websites, boosting in-store search, curating categories as well as
+          photoshop photo editing skills.
+          <br />
+          Over the past year I have moved more into learning and exploring the
+          coding side of eCommerce websites, and am very interested in learning
+          more about front end web and back end web development.
+        </p>
+      </section>
 
-    <div>
-      <h2>Let's Connect</h2>
-      <ul class="connect">
-        <li><a href="mailto:keriwest88@gmail.com?subject=Mail from Website" target="_blank"><strong>Email</strong></a></li>
-        <li><a href="https://github.com/wessstt" target="_blank"><strong>GitHub</strong></a></li>
-        <li><a href="https://www.linkedin.com/in/keri-west/" target="_blank"><strong>LinkedIn</strong></a></li>
-      </ul>
+      <section id="skills">
+        <header>
+          <h4>skills</h4>
+        </header>
+        <ul></ul>
+      </section>
+
+      <section>
+        <header>
+          <h2>&lt; let's connect &gt;</h2>
+        </header>
+
+        <nav>
+          <ul class="connect">
+            <li>
+              <a
+                href="mailto:keriwest88@gmail.com?subject=Mail from Website"
+                target="_blank"
+                ><strong>Email</strong></a
+              >
+            </li>
+            <li>
+              <a href="https://github.com/wessstt" target="_blank"
+                ><strong>GitHub</strong></a
+              >
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/in/keri-west/" target="_blank"
+                ><strong>LinkedIn</strong></a
+              >
+            </li>
+          </ul>
+        </nav>
+      </section>
     </div>
+
+    <footer>
+      <script src="js/index.js"></script>
+    </footer>
```

Comment:
```suggestion
    </footer>
    <script src="js/index.js"></script>
```
I'd suggest keeping the `script` at the very, very end, just before `</body>`.

Reply:
fixed. Thank you.